src/libraries/System.Private.CoreLib/src/Internal/Console.Android.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Internal
+{
+    public static partial class Console
+    {
+        public static unsafe void Write(string s)
```

Comment:
Should this have some conditioned under something like TRACE  or DEBUG?

Reply:
I do not think it should be conditioned. It would just make it less convenient for use (one has to delete the condition to debug a problem in release CoreLib).

People are not expected to leave live calls to this method in shipping code.